tests/integration/cri-containerd/integration-tests.sh

```diff
@@ -257,6 +257,11 @@ function TestContainerMemoryUpdate() {
 
 	if [[ "${KATA_HYPERVISOR}" != "qemu" ]] || [[ "${ARCH}" == "ppc64le" ]] || [[ "${ARCH}" == "s390x" ]]; then
 		return
+	elif [[ "${KATA_HYPERVISOR}" == "qemu-runtime-rs" ]]; then
```

Comment:
This arm cannot be reached since `"${KATA_HYPERVISOR}" != "qemu"` obviously :wink:

If you really want to print the info message you should do this check first. Otherwise you can just drop that hunk.

Reply:
Yeah, you are right. I will update the PR. Thanks!